framework-docs.gradle

```diff
@@ -0,0 +1,218 @@
+plugins {
+	id 'kotlin'
+	id 'io.spring.antora.generate-antora-yml' version '0.0.1'
+	id 'org.antora' version '1.0.0'
+}
+
+description = "Spring Framework Docs"
+
+apply from: "${rootDir}/gradle/publications.gradle"
+
+
+antora {
+	version = '3.2.0-alpha.2'
+	playbook = 'antora-playbook.yml'
+//	playbookProvider {
+//		repository = 'rwinch/spring-framework'
+//		branch = 'docs-build'
+//		path = 'lib/antora/templates/per-branch-antora-playbook.yml'
+//		checkLocalBranch = true
+//	}
+	options = ['--clean', '--stacktrace']
+	environment = [
+			'ALGOLIA_API_KEY': '82c7ead946afbac3cf98c32446154691',
+			'ALGOLIA_APP_ID': '244V8V9FGG',
+			'ALGOLIA_INDEX_NAME': 'framework-docs'
+	]
+	dependencies = [
+			'@antora/atlas-extension': '1.0.0-alpha.1',
+			'@antora/collector-extension': '1.0.0-alpha.3',
+			'@asciidoctor/tabs': '1.0.0-beta.3',
+			'@opendevise/antora-release-line-extension': '1.0.0-alpha.2',
+			'@springio/antora-extensions': '1.1.0',
+			'@springio/asciidoctor-extensions': '1.0.0-alpha.9'
+	]
+}
+
+
+tasks.named("generateAntoraYml") {
+	dependsOn dependencyVersions
+	asciidocAttributes = project.provider( {
+		return  ["spring-version": project.version ]
+	} )
+}
+
+
+jar {
+	enabled = false
+}
+
+javadoc {
+	enabled = false
+}
+
+repositories {
+	maven {
+		url "https://repo.spring.io/release"
+	}
+}
+
+/**
+ * Produce Javadoc for all Spring Framework modules in "build/docs/javadoc"
```

Comment:
For cleanliness, would you mind also changing this and other Framework references to LDAP even if they are in the comments?